util/allWords.js

```diff
@@ -0,0 +1,3727 @@
+export const words = [
```

Comment:
can we put less hard coded words. just keep like 10 words while we build out the UI.

Reply:
Wouldn't it be better to have more words so that we have more of a chance of finding words that may cause problems.

Reply:
We will be using the DB soon to query for words. we don't need to keep all the words in a file.